### PR TITLE
Feature/41100 calendar widget

### PR DESCRIPTION
**Pull request by csumudu**

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/41100

File: example/src/App.tsx [hunk 1]
```diff
@@ -1184,7 +1193,16 @@ class App extends React.Component<{}, State> {
         </div> */}
         <div>
           Pagination:
-          <Pagination {...PaginationDefaultProps} pageSizeList={[3, 6, 9]} showSizeChanger={true} onChange={(page: number) => this.setState({ paginationCurrent: page })} current={this.state.paginationCurrent} total={25} />
+          <Pagination
+            {...PaginationDefaultProps}
+            pageSizeList={[3, 6, 9]}
```
Comment (CaptainSadaf) [inline example/src/App.tsx:1198]: @csumudu Is that page size list hard coded?

File: example/src/App.tsx [hunk 2]
```diff
@@ -1093,6 +1094,14 @@ class App extends React.Component<{}, State> {
       { text: 'How are you?', icon: 'alert' },
     ];
 
+    const events: CalendarEvent[] = [
+      {
+        title: 'Birthday',
+        start: new Date(2020, 9, 6),
+        end: new Date(2020, 9, 7),
```
Comment (CaptainSadaf) [inline example/src/App.tsx:1101]: @csumudu Are these dates are hard coded?

**Review by CaptainSadaf — APPROVED** [review]

File: example/src/App.tsx (hunk 2 above)
Comment (csumudu) [reply example/src/App.tsx:1101]: Yes, it's a example page with dummy data just to test the widget

File: example/src/App.tsx (hunk 1 above)
Comment (csumudu) [reply example/src/App.tsx:1198]: Yes, it's a example page with dummy data just to test the widget

**Pull request merged**